### PR TITLE
Support declaration map in DTE

### PR DESCRIPTION
**Pull request by h-joo**
Support declaration map in DTE

### CODE REVIEW

File: external-declarations/package.json [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
-        "run-tests-parallel": "node ./build/test-runner/parallel-run.js --rootPaths=../tests/cases --libPath=../tests/lib --type=all --shardCount=8 --forceIsolatedDeclarations --outputPath=./tsc-tests/$original  --configFile=./original-tests.jsonc",
-        "run-tests": "node ./build/test-runner/test-runner-main.js --type=all --rootPaths=../tests/cases --outputPath=./tsc-tests/$original --configFile=./original-tests.jsonc",
-        "transform-tests-parallel": "node ./build/code-mod/tsc-test-fixer/run-test-updater-parallel.js --rootPaths=../tests/cases --shardCount=8",
-        "transform-tests": "node ./build/code-mod/tsc-test-fixer/run-test-updater.js --rootPaths=../tests/cases",
-        "run-transformed-tests-parallel": "node ./build/test-runner/parallel-run.js --rootPaths=./tsc-tests/updated-tests --libPath=../tests/lib --type=all --shardCount=8 --outputPath=./tsc-tests/$transformed --configFile=./fixed-tests.jsonc",
-        "run-transformed-tests": "node ./build/test-runner/test-runner-main.js --type=all --rootPaths=./tsc-tests/updated-tests  --libPath=../tests/lib --outputPath=./tsc-tests/$transformed --configFile=./fixed-tests.jsonc",
```
Comment (dragomirtitian) [inline external-declarations/package.json]: I don't think changing these now makes sense. We are working on removing all of this anyway.

Comment (h-joo) [reply external-declarations/package.json]: I removed the commit for the renaming change. Do you want me to revert all changes in this file?

Comment (h-joo) [reply external-declarations/package.json]: I'm highly suspecting that this PR will have a merge conflict with yours, I'll merge after you merge the test updates PR

Comment (dragomirtitian) [reply external-declarations/package.json]: @h-joo yup. There are conflicts. Can you please have a look?

Comment (h-joo) [reply external-declarations/package.json]: Rebased this branch on `isolated-declarations`

**Pull request merged**